themes/default/master.php

```diff
@@ -13,6 +13,7 @@
     <link rel="stylesheet" href="<?= base_url('css/app.css') ?>">
     <script src="https://kit.fontawesome.com/9ce245a629.js" crossorigin="anonymous"></script>
     <?= $this->renderSection('styles') ?>
+    <script src="<?= base_url('js/app.js') ?>"></script>
```

Comment:
It is usually recommended to load file **JS** at the end of the HTML. I was curious to see if there is a special reason for moving?

Reply:
Thank you!